lib/http.c

```diff
@@ -3384,9 +3411,20 @@ static CURLcode http_on_response(struct Curl_easy *data,
     conn->bundle->multiuse = BUNDLE_NO_MULTIUSE;
   }
 
+  if(k->httpcode < 100 && last_hd) {
```

Comment:
isn't `< 100` just outright error or is there some other hidden meaning in this specific case?

Reply:
A bug, thanks for catching. Fixed in last push.